tests/golang/YRoot

```diff
@@ -1,3 +1,21 @@
 ExtDockerImage('scratch', image='scratch')
 
 ExtDockerImage('builder', image='golang', tag='1.14')
+
+CustomInstaller(
+  'protoc-gen-go',
+  uri='',
+  script='install-protoc-gen-go.sh',
+)
+
+DockerImage(
+    'builder-with-proto',
```

Comment:
@itamaro , this image take quite some time to build since it build protobuf. Should we use the base image mechanism you are using in the proto test? can you push it to the same place?
I can create an image which both tests can use but since each test is a different project not sure how to do it in the current structure. WDYT?

Reply:
easiest thing would be to move over this test under `tests/proto` so base image sharing is within the same project.

the other option would be creating a separate test project just for a common base image (that we'll build and push manually), and consume that image in both projects via `ExtDockerImage`.

is there a way to give you permissions to push to my docker hub repo?

Reply:
I've moved the test code from tests/golang to tests/proto and added the go and go protoc plugin into the image used there.
I've change the version and pushed it as ```yabt/proto-builder:3.5.1-go```.
yabt is a team I've created and added you to it.